README.md

```diff
@@ -35,6 +35,9 @@ database:
     username: root
     password: ""
     schema: your_schema
+    # SSL support
+    ssl:
```

Comment:
Maybe better not to include this by default, otherwise users will end up enabling SSL by default.